src/status_im/common/controlled_input/utils.cljs

```diff
@@ -130,3 +136,29 @@
 (defn empty-value?
   [state]
   (string/blank? (:value state)))
+
+(defn- fiat->crypto
+  [value conversion-rate]
+  (-> value
+      (money/fiat->crypto conversion-rate)
+      (wallet-utils/cut-crypto-decimals-to-fit-usd-cents conversion-rate)))
+
+(defn- crypto->fiat
+  [value conversion-rate]
+  (-> value
+      (money/crypto->fiat conversion-rate)
+      (wallet-utils/cut-fiat-balance-to-two-decimals)))
+
+(defn ->crypto
```

Comment:
In components that use controlled input (like send screen) we don't need to maintain a separate value for limit, we can rely on :upper-limit of controlled input, because it is converted along with value.